.eslintignore

```diff
@@ -1,4 +1,5 @@
 build
 public
+coverage
```

Comment:
@juancarlosfarah, I have added this because the `linter` is always testing its content.

Reply:
Sounds good!